concent_api/core/tests/test_core_views.py

```diff
@@ -496,6 +510,98 @@ def test_send_should_return_http_202_if_task_to_compute_deadline_is_correct(self
 
             self.assertIn(response_202.status_code, [200, 202])
 
+    @freeze_time("2017-11-17 10:00:00")
+    def test_send_task_to_compute_without_public_key_should_return_http_400(self):
+        assert StoredMessage.objects.count() == 0
+
+        compute_task_def = message.ComputeTaskDef()
```

Comment:
@rwrzesien, why wouldn't you use `self.task_to_compute`? This boilerplate code seems unnecessary ;)

Reply:
:+1: